runbot_travis2docker/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
```

Comment:
no license in init.py

Reply:
I'm using our template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L2

Should we delete this part from the template?
Feel free of create a new PR to MT in order to delete it, please